common/cards/hermits/orionsound-rare.ts

```diff
@@ -0,0 +1,79 @@
+import {HERMIT_CARDS} from '..'
+import {CardPosModel} from '../../models/card-pos-model'
+import {GameModel} from '../../models/game-model'
+import HermitCard from '../base/hermit-card'
+
+class OrionSoundRareHermitCard extends HermitCard {
+	constructor() {
+		super({
+			id: 'orionsound_rare',
+			numericId: 155,
+			name: 'Ollie',
+			rarity: 'rare',
+			hermitType: 'explorer',
+			health: 260,
+			primary: {
+				name: 'Concert',
+				cost: ['any'],
+				damage: 60,
+				power: null,
+			},
+			secondary: {
+				name: 'Melody',
+				cost: ['explorer', 'explorer'],
+				damage: 0,
+				power: 'Heal all allied afk hermits 30hp. Heal all opposing afk hermits 10hp',
```

Comment:
I think this would be more interesting if it healed *all* afk hermits the same, and maybe also did a bit of damage?